settings.py

```diff
@@ -1033,6 +1033,9 @@
     r'^/a/{domain}/apps/download/',
 ]
 
+# Disable builtin throttling for two factor backup tokens
```

Comment:
The comment at least makes it clear the disabling is intentional, but it might also be nice to explicitly note that we have our own lockout mechanism (and maybe point to it).

Reply:
updated